src/config/user_config.hpp

```diff
@@ -978,7 +979,7 @@ namespace UserConfigParams
                                           "Everything related to online play.") );
 
     PARAM_PREFIX StringUserConfigParam      m_server_multiplayer
-            PARAM_DEFAULT( StringUserConfigParam(   "https://addons.supertuxkart.net/api/",
+            PARAM_DEFAULT( StringUserConfigParam(   "https://online.supertuxkart.net/api/",
```

Comment:
This is problematic because it only changes the *default* value. If the user already has a config file with the previous value, it will not update. Same for the other similar config keys. We need a way to make sure everyone is using the latest server.

A quick and dirty fix, I guess, would be to just rename the config keys. There may be other better ways?

Reply:
IMO these should be in `stk_config.xml`, not the user config.  See e.g. the Donate URL.